app/konnect/org-management/oidc-idp.md

```diff
@@ -46,7 +46,61 @@ authentication.
     * The path does not require a slash (`/`).
 
 1. Click **Save**.
+<!-- copied from okta instructions  as is also relevant to OIDC but needs SME edit for actual details  
```

Comment:
> actual details

question: do we want to walk through enabling and then filling out the team mappings form [here](https://cloud.konghq.tech/global/settings/auth-settings#team-mappings)? I can provide an example walk-through if so.

Reply:
@iamnande I think a walkthrough of team mappings would be great! Thanks!

Reply:
@iamnande Could you provide an example walkthrough of the team mappings form [here](https://cloud.konghq.tech/global/settings/auth-settings#team-mappings)?